components/calendar/calendar.js

```diff
@@ -397,9 +402,13 @@ class Calendar extends LocalizeStaticMixin(LitElement) {
 		`;
 	}
 
-	focus() {
-		const firstButton = this.shadowRoot.querySelector('d2l-button-icon');
-		if (firstButton) firstButton.focus();
+	focus(focusOnDate) {
```

Comment:
`focus()` is a native method that exists on all elements, so I don't feel great about adding a parameter here. Could the logic be that if it's in "dialog" mode, it does one thing and if not does another?

Reply:
Yup that sounds like a much better idea.